src/main/java/com/github/manolo8/darkbot/gui/drawables/DynamicEntitiesDrawer.java

```diff
@@ -131,23 +146,61 @@ private void drawNpcs(MapGraphics mg) {
     }
 
     private void drawPlayers(MapGraphics mg) {
+        Set<Integer> oosMembers = group.getMembers().stream()
+                .map(PartialGroupMember::getId)
+                .collect(Collectors.toSet());
+
+        Color ally = mg.getColor("allies");
+        Color enemy = mg.getColor("enemies");
+        Color groupMember = mg.getColor("group_member");
+
         for (Player player : players) {
+            oosMembers.remove(player.getId());
             boolean isEnemy = player.getEntityInfo().isEnemy();
-            mg.setColor(isEnemy ? "enemies" : "allies");
-            drawEntity(mg, player, 4, false);
 
+            int level = -1;
+            Color color = isEnemy ? enemy : ally;
+            if (group.hasGroup()) {
+                GroupMember member = group.getMember(player);
+                if (member != null) {
+                    color = groupMember;
+                    level = member.getLevel();
```

Comment:
the level sounds pretty irrelevant info for the bot gui, i don't see the utility in it